### PR TITLE
Implement remaining changes in CDC Log format

### PR DESCRIPTION
**Pull request by haaawk**
In https://github.com/scylladb/scylla/pull/5910 @elcallio implemented first part of the new CDC Log format. This PR implements the remaining part. Namely:
1. operation column now contains string representation of operation type instead of meaningless int
2. CDC Log partition key is now a single cdc$stream_id column instead of two columns (cdc$stream_id_1 and cdc$stream_id_2).

Description of the new CDC Log format can be found here: https://docs.google.com/document/d/1riJlkormkzZL0SYqQixxpsDXiURCjG0n_9KJS21zCto/edit

Tests: unit(dev)

### CODE REVIEW

File: cdc/log.cc [hunk 1]
```diff
@@ -63,6 +63,27 @@ using namespace std::chrono_literals;
 
 logging::logger cdc_log("cdc");
 
+sstring cdc::operation_to_text(cdc::operation op) {
+    switch (op) {
```
Comment (elcallio) [inline cdc/log.cc]: const sstring& / const std::string&? Latter could use string literals.

Comment (elcallio) [reply cdc/log.cc]: former constants declared before.

**Comment by elcallio** [comment]
Do we really want op to be a string? Seems expensive, just to make cqlsh look better. It will hardly make code reading the cdc log easier - you instead have to keep a secondary table of string constants to match against. Not easier than int constants. 
Can't we just either invent cql::enum or just augment cqlsh if the latter is the problem?

Other than that, I have no objections.

**Comment by haaawk** [comment]
> Do we really want op to be a string? Seems expensive, just to make cqlsh look better. It will hardly make code reading the cdc log easier - you instead have to keep a secondary table of string constants to match against. Not easier than int constants.
> Can't we just either invent cql::enum or just augment cqlsh if the latter is the problem?
> 
> Other than that, I have no objections.

@avikivity the change is on your request. Please comment.

File: cdc/log.cc (hunk 1 above)
Comment (haaawk) [reply cdc/log.cc]: Done.

Comment (elcallio) [reply cdc/log.cc]: or simply std::string_view by value. Guaranteed least code and least overhead (unless gcc is completely useless).

Comment (haaawk) [reply cdc/log.cc]: Changed to string_view.

**Comment by haaawk** [comment]
I uploaded the non-controversial part of the PR here -> https://github.com/scylladb/scylla/pull/5950

Please review and hopefully merge while I'm checking computed columns for the part 2.

**Comment by nyh** [comment]
@haaawk I don't understand if this PR is still relevant or you split it?

**Comment by haaawk** [comment]
I split this and you seemed to merged the other version so I'm closing this.

**Pull request closed without merging**